Mathlib/Order/Hom/Basic.lean

```diff
@@ -1272,6 +1277,11 @@ end BoundedOrder
 
 end LatticeIsos
 
+-- See note [lower instance priority]
+instance (priority := 100) OrderIsoClass.toOrderIsoClassOrderDual [LE α] [LE β]
+    [EquivLike F α β] [OrderIsoClass F α β] : OrderIsoClass F αᵒᵈ βᵒᵈ where
```

Comment:
Here however `α` and `β` are not out-params in `OrderIsoClass`? @Vierkantor, is this an oversight?